### PR TITLE
Toggle osquery verbose flag on startup

### PR DESCRIPTION
**Pull request by RebeccaMahany**
Capture as many osquery logs on startup as we can in hopes of gathering more info for the monterey bug.

### CODE REVIEW

File: pkg/osquery/extension.go [hunk 1]
```diff
@@ -515,13 +521,63 @@ func (e *Extension) generateConfigsWithReenroll(ctx context.Context, reenroll bo
 		return e.generateConfigsWithReenroll(ctx, false)
 	}
 
+	// Adjust osqueryVerbose value if needed
+	e.osqueryVerboseMutex.Lock()
+	defer e.osqueryVerboseMutex.Unlock()
+	verboseChanged := false
+	if !reenroll && !e.osqueryVerbose {
```
[the remark this answers is not in the document]
Comment (RebeccaMahany) [reply pkg/osquery/extension.go:528]: I think yes, as long as we're fine with it happening on every startup and not just on first-time enrollment?

[the remark this answers is not in the document]
Comment (RebeccaMahany) [reply pkg/osquery/extension.go]: Updated with 👆 in mind, let me know what you think

**Review by directionless — APPROVED** [review]
okay, let's try

File: pkg/osquery/extension.go [hunk 2]
```diff
@@ -515,13 +516,50 @@ func (e *Extension) generateConfigsWithReenroll(ctx context.Context, reenroll bo
 		return e.generateConfigsWithReenroll(ctx, false)
 	}
 
+	// If osquery has been running successfully for 20 minutes, then turn off verbose logs.
+	osqueryVerbose := true
+	if uptimeMins, err := history.LatestInstanceUptimeMinutes(); err == nil && uptimeMins >= 20 {
+		level.Debug(e.logger).Log("msg", "osquery has been up for more than 20 minutes, turning off verbose logging", "uptime_mins", uptimeMins)
```
Comment (directionless) [inline pkg/osquery/extension.go:522]:
```suggestion
	// If osquery has been running successfully for 10 minutes, then turn off verbose logs.
	osqueryVerbose := true
	if uptimeMins, err := history.LatestInstanceUptimeMinutes(); err == nil && uptimeMins >= 10 {
		level.Debug(e.logger).Log("msg", "osquery has been up for more than 10 minutes, turning off verbose logging", "uptime_mins", uptimeMins)
```

**Review by directionless — APPROVED** [review]

**Pull request merged**